src/Detector/Class1.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace Detector
```

Comment:
remove?

Reply:
Sure can remove it. I just left it like this hoping Stella will update it's contents anyway.